### PR TITLE
Collaboration Journey Documentation

### LINKED ISSUE #38: Documentation on collaboration journey from developers' perspectives
- [x] How to get onboarded as a developer
- [x] How developers work together (across company)

can add other context if necessary

### PR DESCRIPTION
**Pull request by ayogasekaram**
Added both developer and PO collaboration journey. Closes #38 and #39

### CODE REVIEW

**Comment by edelarua** [comment]
Hi @ayogasekaram,
I restructured/combined the two sections you made and moved them to an "About" page - feel free to edit/change what I've done as it's just a suggestion, I just wasn't sure how to relay my suggestions without creating a commit. As of now I haven't looked at any of the text yet, just the layout.

**Comment by ayogasekaram** [comment]
@edelarua great suggestion! I found the topics too sparse to be a stand alone page so the about section sounds appropriate. I'll read through the text to make sure it's not repeating information already on this page.

File: index.qmd [hunk 1]
```diff
@@ -29,7 +29,7 @@ The falcon initiative is an industry collaborative effort under pharmaverse that
 </div> 
 
 <div class="divButton">
-<button onclick="window.location.href = 'quarto/our_journey.html';" class="button">Our Journey</button>
+<button onclick="window.location.href = 'https://pharmaverse.github.io/falcon/quarto/about.html#our-collaboration-journey';" class="button">Our Collaboration Journey</button>
```
Comment (mattkorb) [inline index.qmd:32]:
```suggestion
<button onclick="window.location.href = '/quarto/about.html#our-collaboration-journey';" class="button">Our Collaboration Journey</button>
```

Comment (mattkorb) [reply index.qmd:32]: I think this should be sufficient

File: quarto/about.qmd [hunk 2]
```diff
@@ -2,4 +2,31 @@
...
 ---
 
-Page content coming soon!
+The falcon project is a product of industry collaborations to encourage the development of shared code, expertise, knowledge and products.
+
+In the near future, the goal is to develop functionality to create regulatory reporting tables suitable for FDA filing. The long-term objective of this collaboration is to develop an R package that can be used industry wide - developing a cross-industry standard and reducing manual work from a reporting perspective.
+
+## Our Collaboration Journey
+
+Current collaborators are from Roche, Sanofi, Boehringer-Ingelheim, and Moderna. Developers are connected via GitHub for the creation of shared solutions using publicly available analysis packages for R. Developers are invited to browse the falcon project repository and related packages for insights into our template designs.
```
Comment (mattkorb) [inline quarto/about.qmd:11]:
```suggestion
Current collaborators are from Roche, Sanofi, Boehringer-Ingelheim, and Moderna. Developers are connected via [GitHub ](https://github.com/pharmaverse/falcon) for the creation of shared solutions using publicly available analysis packages for R. Developers are invited to browse the falcon project repository and related packages for insights into our template designs.
```

File: quarto/about.qmd [hunk 3]
```diff
@@ -2,4 +2,31 @@
...
+
+### Joining as a Collaborator
+
+We invite further collaboration with additional companies joining this initiative and contributing to the falcon project. Please feel free to contact any of the repository maintainers for additional information and check out the section below for more information on joining the project as a developer.
+
+#### Information for Developers
+
+As a pharmaverse collaboration, we invite any interested developers to join the falcon team and contribute to template development. To on-board as a developer, please reach out to a product owner (Vincent Shen, Freeman Wang, Lian Lin or Thomas Neitman) to receive access to the falcon GitHub repository.
```
Comment (mattkorb) [inline quarto/about.qmd:19]: Maybe also link the email address of the PO's?
Should we also mention Juergen Boehl as PO and is Thomas still involved? I thought he left Roche :)

And we could also link to the GitHub repo again.

**Review by mattkorb — COMMENTED** [review]
Hi @ayogasekaram , 
looks good to me, I have only some minor suggestions. 
Please let me know what you think about it. :)

**Review by mattkorb — APPROVED** [review]
Hi @ayogasekaram 
Looks good to me, thank you for your work. :)

**Pull request merged**